position/position.gno

```diff
@@ -269,17 +280,16 @@ func increaseLiquidity(params IncreaseLiquidityParams) (uint64, *u256.Uint, *u25
 	checkDeadline(params.deadline)
 
 	position := positions[params.tokenId]
-	liquidity, amount0, amount1 := addLiquidity(
-		AddLiquidityParams{
-			poolKey:        position.poolKey,
-			tickLower:      position.tickLower,
-			tickUpper:      position.tickUpper,
-			amount0Desired: params.amount0Desired,
-			amount1Desired: params.amount1Desired,
-			amount0Min:     params.amount0Min,
-			amount1Min:     params.amount1Min,
-		},
+	ll := newAddLiquidtyParams(
```

Comment:
`ll` seems like bad naming

Reply:
changed to `liqParams`

https://github.com/gnoswap-labs/gnoswap/pull/296/commits/299d6f8bd76599de0ad674ee87784fc7ccfe8f9f